built_value_generator/lib/src/value_source_class.dart

```diff
@@ -39,7 +40,14 @@ abstract class ValueSourceClass
       name.startsWith('_') ? '_\$${name.substring(1)}' : '_\$$name';
 
   @memoized
-  ClassElement get builderElement => element.library.getType(name + 'Builder');
+  ClassElement get builderElement {
+    final result = element.library.getType(name + 'Builder');
+    if (result == null) return null;
+    // If the builder is in a generated file, then we're analyzing _after_ code
```

Comment:
line too long

Reply:
This also looks like 80 cols--should be okay.